config/webpack.config.js

```diff
@@ -400,6 +404,9 @@ module.exports = function (env, ilibAdditionalResourcesPath) {
 						mangle: {
 							safari10: true
 						},
+						// Added for profiling in devtools
```

Comment:
Which PR related to this?

Reply:
It's CRA`s #7737 PR.
While profiling in React, via using react-dom/profiling, it needs to keep its class name & function name.
These options help to keep names, but these options are for React, CRA.
No needs I think.

Reply:
Please remove them.

Reply:
OK